controllers/controllers.go

```diff
@@ -50,10 +50,26 @@ func (r *instanceReconciler) configureInstance(instance *instances.InstanceInfo,
 	if err != nil {
 		return errors.Wrap(err, "failed to create new nodeconfig")
 	}
-	if err := nc.Configure(); err != nil {
-		return errors.Wrap(err, "failed to configure Windows instance")
+
+	// If the given instance has a Node already, check which version of WMCO configured it
+	if instance.Node != nil {
```

Comment:
why not check instance.Node == nil and configure first to reduce indentation.

Reply:
See https://github.com/openshift/windows-machine-config-operator/pull/513#discussion_r668811315 as to why I changed it from that

Reply:
My [comment](https://github.com/openshift/windows-machine-config-operator/pull/513#discussion_r668292078) still stands even with this change. Or am I missing something?